README.md

```diff
@@ -4,14 +4,48 @@ Generates Rust messages from a `dbc` file.
 
 ⚠️ This is experimental - use with caution. Breaking changes will happen when you least expect it. ⚠️
 
-# Usage
+## Usage
 
 Generate `messages.rs` from `example.dbc`.
 ```bash
 cargo run -- testing/dbc-examples/example.dbc dir/where/messages_rs/file/is/written
 ```
 
-# Development
+If some field name starts with a non-alphabetic character or is a Rust keyword then it is prepended with `x`.
+
+For example:
+```
+VAL_ 512 Five 0 "0Off" 1 "1On" 2 "2Oner" 3 "3Onest";
+```
+..is generated to:
+```rust
+pub enum BarFive {
+    X0off,
+    X1on,
+    X2oner,
+    X3onest,
+    Other(bool),
+}
+```
+
+`Type` here:
+```
+SG_ Type : 30|1@0+ (1,0) [0|1] "boolean" Dolor
+```
+..would become Rust keyword `type` therefore it is prepended with `x`:
+```rust
+pub fn xtype(&self) -> BarType {
+    match self.xtype_raw() {
+        false => BarType::X0off,
+        true => BarType::X1on,
+        false => BarType::X2oner,
+        false => BarType::Type,
```

Comment:
Uhh it looks like a bug to match on false 3 times. Does this example (also used in tests) actually make sense in a real scenario?

Reply:
Hah, interesting - how the hell this thing actually built.

Reply:
This bug is introduced here: https://github.com/technocreatives/dbc-codegen/commit/ee84b9bae04889a1c67b412c2703ac0e720a3dd7#diff-db06603f82c0fa96425947567eb2a093f55f85cc496b1a569150391717088a97R367-R374

Reply:
Wait a sec, `dbc` file itself is incorrect.

Reply:
This is `boolean` - min is `0` and max is `1`.
```
SG_ Type : 30|1@0+ (1,0) [0|1] "boolean" Dolor
```

However I defined it with 4 values:
```
VAL_ 512 Type 0 "0Off" 1 "1On" 2 "2Oner" 3 "TYPE";
```

Reply:
This is now fixed.